staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -162,6 +162,31 @@ func updatePod(t *testing.T, s storage.Interface, obj, old *example.Pod) *exampl
 	return result
 }
 
+func TestCreate(t *testing.T) {
+	server, etcdStorage := newEtcdTestStorage(t, etcd3testing.PathPrefix())
+	defer server.Terminate(t)
+	cacher, _, err := newTestCacher(etcdStorage)
+	if err != nil {
+		t.Fatalf("Couldn't create cacher: %v", err)
+	}
+	defer cacher.Stop()
+
+	checkObject := func(context.Context, *testing.T, storage.Interface, string) {}
```

Comment:
with a no-op checkObject, are we actually verifying anything got stored?

Reply:
We do - this is just an additional check - see my comment above.